### PR TITLE
Davids branch

### PR DESCRIPTION
**Pull request by dsolowitz**
Closes issue#111

### CODE REVIEW

File: src/hooks/useTableColumns.js [hunk 1]
```diff
@@ -31,7 +34,12 @@ export const useTableColumns = () => {
     () => [
       {
         Header: 'Name',
-        accessor: 'candidate_full_name',
+        accessor: ({ candidate_full_name, committee_sboe_id }) => (
+          <Link to={CANDIDATE_URL + committee_sboe_id}>
+            {' '}
```
Comment (Dafondo) [inline src/hooks/useTableColumns.js:39]: I think you can delete this. I'm guessing this space was generated by the auto-formatter.

**Review by Dafondo — APPROVED** [review]
Looks good to me

File: src/hooks/useTableColumns.js (hunk 1 above)
Comment (mcelestini) [reply src/hooks/useTableColumns.js:39]: agreed, let's nix this. If we need some spacing, let's use CSS with margin

**Review by mcelestini — APPROVED** [review]
💯 looks great

**Pull request merged**